api/api_ethereum.go

```diff
@@ -1364,6 +1364,9 @@ func (api *EthereumAPI) FillTransaction(ctx context.Context, args EthTransaction
 // SendRawTransaction will add the signed transaction to the transaction pool.
 // The sender is responsible for signing the transaction and using the correct nonce.
 func (api *EthereumAPI) SendRawTransaction(ctx context.Context, input hexutil.Bytes) (common.Hash, error) {
+	if len(input) == 0 {
+		return common.Hash{}, fmt.Errorf("EOF")
```

Comment:
I think using the phrase "Empty input" would indeed make it clearer. How do you think?

Reply:
I set the error message to EOF for unity with the klay namespace, but this idea looks better